### PR TITLE
Upgrade dependencies #553

### PR DESCRIPTION
**Pull request by HDouss**
Resolving #553 
* upgrade dependencies
* enforce jcabi-aspects version for qulice-xml (see issues jcabi/jcabi-aspects#194 and jcabi/jcabi-xml#73)
* qulice 0.14 compliance
* force flushing writer in PMD tests before match

### CODE REVIEW

**Comment by krzyk** [comment]
@HDouss AFAIR we are at ASM 5 already, it was required by Findbugs, dies the new invoker still fail?

**Comment by davvd** [comment]
@hdouss Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@pinaf it's yours, please review

**Comment by HDouss** [comment]
@krzyk The build does not fail. I was confused by "exception in runtime". But you are right, we are already at ASM 5. So I removed hardcoded version in last commit 9f718a153ffa3f94ac004d97c3f4ad3c6647661e

File: qulice-xml/src/main/java/com/qulice/xml/XmlValidator.java [hunk 1]
```diff
@@ -53,6 +53,7 @@
  *
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
+ * @checkstyle IndentationCheck (5 lines)
```
Comment (pinaf) [inline qulice-xml/src/main/java/com/qulice/xml/XmlValidator.java]: @HDouss have you something like this?
```java
@SuppressWarnings({ "PMD.AvoidInstantiatingObjectsInLoops",
    "PMD.ExceptionAsFlowControl" })
```

**Comment by pinaf** [comment]
@HDouss good job. 1 comment.

File: qulice-xml/src/main/java/com/qulice/xml/XmlValidator.java (hunk 1 above)
Comment (HDouss) [inline qulice-xml/src/main/java/com/qulice/xml/XmlValidator.java]: @pinaf I don't understand your comment. May be I'm wrong but I think the indentation above is correct. In contrary, I think the code in your comment doesn't comply (lcurly should be on a new line, the same for the right parenthesis). What do you think?

**Comment by HDouss** [comment]
@pinaf I got it. Thanks !

**Comment by pinaf** [comment]
@rultor merge

**Comment by pinaf** [comment]
@HDouss I mentioned it because there are multiple examples of this indentation in other projects.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by krzyk** [comment]
@rultor deploy pls

**Comment by davvd** [comment]
@pinaf **19 mins** sent to your balance (ID `AP-7FL84732H9013415V`), many thanks! It took 43 hours and 32 mins.; you're getting extra minutes here (c=4); +19 added to your rating, at the moment it is: [+5004](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor deploy pls

**Pull request merged**